### PR TITLE
Alt-click hotkeys for many items

### PR DESCRIPTION
**Pull request by 9600bauds**
I'm not 100% sure about the amount of sanity that this requires.

### CODE REVIEW

**Comment by PJB3005** [comment]
:+1: :x: :100:

**Comment by ihadtoregisterforthis** [comment]
Enlighten the people illiterate at code what items are included by the denomination "many", plox

**Comment by 9600bauds** [comment]
- rscadd: Alt-clicking a reagent container (i.e. beaker) while it's in your inventory will now directly bring up the \"Set transfer amount\" menu.
-  - rscadd: Alt-clicking a Chemistry Dispenser will now directly eject it's beaker, if any.
-  - rscadd: Alt-clicking a portable scrubber or portable pump will now eject their held tank, without having to open their menu.
-  - rscadd: Alt-clicking a washing machine will now start it. Likewise for Centrifuges.
-  - rscadd: Janicarts have now gained a \"Remove Trash Bag\" verb. Alt-clicking a Janicart will directly remove it's trash bag. Clicking a Janicart while you're riding it will no longer bring up a popup asking what you want to do, it will instead always make you stop riding it. If you want to remove the trash bag while riding it, use the verb or alt-click.
-  - experiment: If you can think of any other alt-click shortcuts that you'd like to see in the game, please don't hesitate to use the in-game bug reporter to make a feature request.

**Comment by ihadtoregisterforthis** [comment]
> > > > > > please don't hesitate to use the in-game bug reporter to make a feature request.

**Comment by 9600bauds** [comment]
Precisely

**Comment by ghost** [comment]
:+1:

**Comment by Konater** [comment]
Sleepers and DNA scanners too please

**Comment by 9600bauds** [comment]
You can already eject patients with clickdrag, unless you mean something else

File: code/game/machinery/atmoalter/portable_atmospherics.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 			update_icon()
 
 /obj/machinery/portable_atmospherics/process()
-	if(!connected_port) //only react when pipe_network will ont it do it for you
+	if(!connected_port) //only react when pipe_network will not it do it for you
```
Comment (clusterfack) [inline code/game/machinery/atmoalter/portable_atmospherics.dm]: autism speaks, it's time to listen

File: html/changelogs/9600bauds_alt.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+changes: 
+  - rscadd: Alt-clicking a reagent container (i.e. beaker) while it's in your inventory will now directly bring up the \"Set transfer amount\" menu.
+  - rscadd: Alt-clicking a Chemistry Dispenser will now directly eject it's beaker, if any.
+  - rscadd: Alt-clicking a portable scrubber or portable pump will now eject their held tank, without having to open their menu.
+  - rscadd: Alt-clicking a washing machine will now start it. Likewise for Centrifuges.
+  - rscadd: Janicarts have now gained a \"Remove Trash Bag\" verb. Alt-clicking a Janicart will directly remove it's trash bag. Clicking a Janicart while you're riding it will no longer bring up a popup asking what you want to do, it will instead always make you stop riding it. If you want to remove the trash bag while riding it, use the verb or alt-click.
+  - experiment: If you can think of any other alt-click shortcuts that you'd like to see in the game, please don't hesitate to use the in-game bug reporter to make a feature request.
```
Comment (clusterfack) [inline html/changelogs/9600bauds_alt.yml]: BAN

**Comment by 9600bauds** [comment]
Fak u, player feedback is important and there's a serious problem with NOBODY EVER OPENING ISSUES

**Comment by clusterfack** [comment]
There is also a serious problem with BEING FLOODED WITH SHITTY ISSUES THANKS TO PLAYERS WHO ARE SO MINIMALLY CHALLENGED

**Comment by Zth--** [comment]
Yeah bauds is right, we have almost no issues, give the coders something to do

**Comment by ihadtoregisterforthis** [comment]
## ̞͉̗͒ͨͥ̓̆̑:̰̜̖ͥ̿̑̃͑:°)

**Comment by 9600bauds** [comment]
Guess map rotation bringing more issues opened got into everyone's nerves here

File: code/game/machinery/atmoalter/portable_atmospherics.dm (hunk 1 above)
Comment (9600bauds) [inline code/game/machinery/atmoalter/portable_atmospherics.dm:30]: I recall a line from you in the IRC shittalking people who tiptoe around awful stuff in the code and leave it unfixed

Comment (clusterfack) [inline code/game/machinery/atmoalter/portable_atmospherics.dm]: Pretty sure I wasn't talking about correcting typos in comments

Comment (9600bauds) [inline code/game/machinery/atmoalter/portable_atmospherics.dm:30]: It's awful to me and I stumbled upon it. I can't foresee any conflicts, future maintainability, or anything actually bad other than being called an autist from taking the 0-2 seconds to fix it.

**Comment by 9600bauds** [comment]
Re-consider DNM

**Comment by ihadtoregisterforthis** [comment]
Feature requests are awwright, using the bug reporter for them is uncool.

**Comment by 9600bauds** [comment]
It's not even a bug reporter, it's a web reporter. A lot of massive pitfalls in how the game plays out fall technically into "feature requests". Heck, even map issues are "feature requests".
And I already fixed the changelog

**Comment by Zth--** [comment]
I would like alt click to exit from sleeper.

**Comment by 9600bauds** [comment]
You can clickdrag out of it, can't you?

**Pull request merged**